app/views/content/international-qualified-teacher-status/_after-accordion.md

```diff
@@ -4,12 +4,21 @@ Learn more about [iQTS policy](https://www.gov.uk/government/publications/intern
 
```

Comment:
Not a comment on this line but that on line 3 (can't comment on that!) - Teachers' Standards link must have "iQTS" in it for consistency/accessibility and to differentiate it from QTS standards